packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -51,7 +51,9 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
   const [jobTitle, setJobTitle] = useState<Maybe<string>>()
   const [image, setImage] = useState<Maybe<ImageRefFragment>>()
   const [bio, setBio] = useState<RichTextBlockValue>(createDefaultValue())
-  const [links, setLinks] = useState<ListValue<AuthorLink>[]>([])
+  const [links, setLinks] = useState<ListValue<AuthorLink>[]>([
+    {id: generateID(), value: {title: '', url: ''}}
```

Comment:
@HanaAyman why did you change the default value?

Reply:
That's what makes it that there is always by default an empty input field for the links.